paddle/fluid/imperative/layer.cc

```diff
@@ -120,6 +168,36 @@ class Autograd {
   }
 };
 
+VarBase* VarBase::NewVarBase(const platform::Place& dst_place,
```

Comment:
return a unique pointer and release in pybind?

Reply:
Done